go.mod

```diff
@@ -47,7 +47,7 @@ require (
 	github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4 // indirect
 	github.com/99designs/keyring v1.2.1 // indirect
 	github.com/ChainSafe/go-schnorrkel v1.0.0 // indirect
-	github.com/CosmWasm/wasmvm v1.2.4 // indirect
+	github.com/CosmWasm/wasmvm v1.5.0 // indirect
```

Comment:
It should be changed wasmvm v1.5.1
https://github.com/CosmWasm/advisories/blob/main/CWAs/CWA-2023-004.md